agent/bls.mjs

```diff
@@ -482,25 +514,26 @@ function perform_bls_verify_i(
 ) {
     if( !joResultFromNode )
         return true;
+    const imaState = state.get();
     const strLogPrefix = cc.bright( strDirection ) + cc.debug( "/" ) + cc.info( "BLS" ) + cc.debug( "/" ) + cc.notice( "#" ) + cc.bright( nZeroBasedNodeIndex ) + cc.debug( ":" ) + " ";
     const nThreshold = discover_bls_threshold( imaState.joSChainNetworkInfo );
     const nParticipants = discover_bls_participants( imaState.joSChainNetworkInfo );
-    const strPWD = shell.pwd();
+    // const strPWD = shell.pwd();
     const strActionDir = alloc_bls_tmp_action_dir();
     const fnShellRestore = function() {
-        shell.cd( strPWD );
+        // shell.cd( strPWD );
         shell.rm( "-rf", strActionDir );
     };
     let strOutput = "";
     try {
-        shell.cd( strActionDir );
+        // shell.cd( strActionDir );
         details.write( strLogPrefix + cc.debug( "BLS node " ) + cc.notice( "#" ) + cc.info( nZeroBasedNodeIndex ) + cc.debug( " - first message nonce is " ) + cc.info( nIdxCurrentMsgBlockStart ) + "\n" );
         details.write( strLogPrefix + cc.debug( "BLS node " ) + cc.notice( "#" ) + cc.info( nZeroBasedNodeIndex ) + cc.debug( " - first source chain name is " ) + cc.info( strFromChainName ) + "\n" );
         details.write( strLogPrefix + cc.debug( "BLS node " ) + cc.notice( "#" ) + cc.info( nZeroBasedNodeIndex ) + cc.debug( " - messages array " ) + cc.j( jarrMessages ) + "\n" );
```

Comment:
Please, modify duplicated code of perform_bls_verify_i and perform_bls_verify_i_u256, according to https://github.com/skalenetwork/internal-support/blob/main/docs/wiki/code-guidelines.md#coherent-style